### PR TITLE
Improve .environment when installing nvm manually

### PR DESCRIPTION
**Pull request by hanoii**

### CODE REVIEW

File: src/languages/nodejs-nvm.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ hooks:
 And in a `.environment` file in the root of your project:
  
 ``` 
+# This is necessary for nvm to work
+unset NPM_CONFIG_PREFIX
+# Disable npm update notifier, being a read only system it will probably annoy you.
```
Comment (Crell) [inline src/languages/nodejs-nvm.md]: Semi-colon, not comma. :smile:

Comment (Crell) [reply src/languages/nodejs-nvm.md]: Also, comment lines should end in a period.

**Comment by Crell** [comment]
I think the build issue here is a transient Node issue.  I've seen it before.  Another commit to fix the comments as noted above will probably clear that up, too.

Thanks, @hanoii!  Does this resolve the issue you were seeing before?

**Comment by hanoii** [comment]
@Crell you mean what I brought up in Slack  yesterday? No, not really. It improved it when `.environment` is sourced, but I still think it's not on build/deploy/command start hooks.

Btw as you brought this up here, just linking the ticket for cross reference.

**Comment by hanoii** [comment]
I just realised that I was modifying a file that's not longer referenced

**Comment by hanoii** [comment]
Slightly modified other small bits.

**Comment by Crell** [comment]
Thanks, @hanoii!

**Pull request merged**